Extension/.scripts/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+    "compilerOptions": {
+        "target": "ES2022",
+        "module": "CommonJS",
```

Comment:
I'm getting an error message on this line: "Option 'module' must be set to 'Node16' when option 'moduleResolution' is set to 'Node16'.",

Reply:
I am not seeing that error at all.

Reply:
Yeah, I don't get it now, but I was seeing it on Friday -- maybe the error appears after running some command or it got fixed? I'm looking into that...

Reply:
I reproed the error in tsconfig -- it doesn't appear initially, but something triggers them to appear "later on" -- not sure yet what exactly.

Reply:
I figured it out -- the error appears when using the VS Code default TS version of 5.2.0, but it goes away after switching to the workspace version of 5.1.6 -- is that okay? Or should something be done in advance to fix the error for 5.2.0?